source

```diff
@@ -102691,10 +102700,12 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    state</span>. Emit the current tag token.</dd>
 
    <dt>EOF</dt>
-   <dd><span>Parse error</span>. Emit an end-of-file token.</dd>
+   <dd><span data-x="parse-error-eof-in-tag">eof-in-tag</span><span>parse
+   error</span>. Emit an end-of-file token.</dd>
 
    <dt>Anything else</dt>
-   <dd><span>Parse error</span>. <span>Reconsume</span> in the <span>before attribute name
+   <dd>This is a <dfn data-x="parse-error-self-closing-start-tag">self-closing-start-tag</dfn>
```

Comment:
Self closing start tags are perfectly fine in foreign content (e.g SVG) or in case of void elements (e.g `<br/>`). The issue here is that instead of closing such tag with `>` it continued with some content, it's usually a sign that you forgot to put `>`, e.g.:
```html
<meta /Lorem ipsum
```
So it's `abruption-of-tag self-closure`